vagrant/base.sh

```diff
@@ -36,8 +36,8 @@ fetch_jdk_tgz() {
   fi
 }
 
-JDK_MAJOR="${JDK_MAJOR:-8}"
-JDK_FULL="${JDK_FULL:-8u202-linux-x64}"
+JDK_MAJOR="${JDK_MAJOR:-11}"
+JDK_FULL="${JDK_FULL:-11.0.2-linux-x64}"
```

Comment:
Temp fix until #17576 gets merged - I'll revert this if the tests pass and I can merge